### DISCUSSION TITLE
Feature request - additional options for recurring transaction frequency

### DISCUSSION THREAD

**Discussion by BackedUpBooty** [firefly-iii/firefly-iii#7481] [General]
Current options for setting up a recurring transaction are currently limited to daily, weekly, every x date of a month, every x day in a month, or yearly. 

![image](https://github.com/firefly-iii/firefly-iii/assets/53868636/1d27f2c5-879c-4cc3-ac60-3da4b7818445)

I understand that these may be the most common, however are there any plans to provide 'every x weeks' or 'every x days' options? I.e. 'every 4 weeks' or 'every 5 days'?

**Comment by JC5** [comment]
That can be accomplished with the "skip" field.
